### DISCUSSION TITLE
会议论文集参考文献引用格式问题

### DISCUSSION THREAD

**Discussion by NELFnhm** [sjtug/SJTUThesis#676] [提问] [answered]
使用交大模板引用会议论文集中析出的文献时，使用@inproceedings引用出的条目中，论文集名称前会带有 “见：”，请问如何去掉“见：”呢？（使用overleaf）

具体条目样式：郑能恒,蔡毅,李霞,等.基于非负矩阵分解和向量相似测度的语音与音乐分离算法[C].见:第十一届全国人机语音通讯学术会议论文集(一).中国陕西西安, 2011

**Comment by skyzh** [comment]
SJTUThesis 文献管理使用 biblatex-gb7714-2015 包，应该是符合国家标准的格式。如果的确有这个需求，您可以修改 https://github.com/sjtug/SJTUThesis/blob/master/texmf/tex/latex/biblatex-gb7714-2015/gb7714-2015.bbx 文件，把 `\def\str@incn{见}` 设置成空字符串。

**Comment by skyzh** [comment, marked as answer]
SJTUThesis 文献管理使用 biblatex-gb7714-2015 包，应该是符合国家标准的格式。如果的确有这个需求，您可以在 `\begin{document}` 之前设定：

```
\DefineBibliographyStrings{english}{
  incn={},
}
```

**Reply by skyzh** [reply]
<img width="466" alt="image" src="https://user-images.githubusercontent.com/4198311/120590709-8bbf9500-c46d-11eb-8732-9b9824b49672.png">

**Reply by NELFnhm** [reply]
非常感谢！那请问英文文献也有一个“in:”需要去掉应该怎么操作呢？

**Reply by NELFnhm** [reply]
我找到了biblatex-gb7714-2015的文档，解决方法是在`incn={},`后添加`in={},`，没有其他问题了，谢谢！

**Comment by hushidong** [comment]
国标文档，见：
http://www.cessp.org.cn/uploads/1/file/public/201607/20160708142456_8mqgu0dpgk.pdf

国标的复现，见文档：
https://github.com/hushidong/biblatex-gb7714-2015/blob/master/tool/stdgbT7714-2015.pdf
